internal/saucecloud/cloud.go

```diff
@@ -64,6 +64,12 @@ type CloudRunner struct {
 	NPMDependencies []string
 
 	interrupted bool
+	CachedData  CachedData
+}
+
+type CachedData struct {
+	VDCBuildURL string
+	RDCBuildURL string
```

Comment:
The `Data` part didn't seem to add much more information, so I'd recommend dropping it (brevity can increase readability)

```suggestion
	Cache  Cache
}

type Cache struct {
	VDCBuildURL string
	RDCBuildURL string
```